packages/embark/src/lib/modules/blockchain_connector/index.js

```diff
@@ -399,7 +399,7 @@ class BlockchainConnector {
       'get',
       '/embark-api/blockchain/transactions/:hash',
       (req, res) => {
-        self.getTransactionByRawTransactionHash(req.params.hash, (err, transaction) => {
+        self.getTransactionByHash(req.params.hash, (err, transaction) => {
```

Comment:
I know @PascalPrecht changed this to `getTransactionByRawTransactionHash` for a specific reason, but don't remember by